ansible/roles/test/files/ptftests/fdb_test.py

```diff
@@ -79,7 +84,7 @@ def check_route(self, src_mac, dst_mac, src_port, dst_port):
                                 eth_type=0x1234)
         self.log("Send packet " + str(src_mac) + "->" + str(dst_mac) + " from " + str(src_port) + " to " + str(dst_port) + "...")
         send(self, src_port, pkt)
-        verify_packet(self, pkt, dst_port)
+        verify_packet_any_port(self, pkt, [dst_port])
```

Comment:
Just curios, what is the difference here?

Reply:
The function name doesn't not reflecting all it does. With the new function, we verify that the packet is received on the expected port and NOT received from any other ports.